### PR TITLE
LPD-1544 Verify if the input date and time value match the correct format

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPD-1544](https://liferay.atlassian.net/browse/LPD-1544)

### CODE REVIEW

**Comment by vngoliveira** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
Hey @vngoliveira, I noticed that your changes were only in DDM. Could you close this PR and open another one in the forms repository?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js
Comment (igor-franca) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js:1]: Hi @vngoliveira, I noticed that your changes were made only in DDM.  Could you close this PR and open another one in the Forms repository.

**Review by igor-franca — CHANGES_REQUESTED** [review]

**Comment by vngoliveira** [comment]
@igor-franca https://github.com/liferay-forms/liferay-portal/pull/2481

**Pull request closed without merging**